src/main/scala/is/hail/variant/GenotypeStream.scala

```diff
@@ -116,6 +128,14 @@ object GenotypeStream {
   }
 }
 
+object ToIteratorGT {
+  def apply(it: Iterable[Genotype]): Iterator[Int] =
```

Comment:
I agree, I've moved it to implicit. And in order to make hardCallIterator return an IntIterator that is not a HardcallGenotypeStreamIterator, I created a HardCallIterator.  Back to you.